src/main/kotlin/baseball/InputManager.kt

```diff
@@ -0,0 +1,37 @@
+package baseball
+
+// 유효성 검사(사용자 숫자)
+fun isValidateInputStringForGame(input: String) {
+    // 빈 문자열 확인
+    if (input.isBlank()) throw IllegalArgumentException("input string is empty")
+    // 3자리 수 길이 확인
+    if (input.length != 3) throw IllegalArgumentException("input string's length is not suitable")
+    // 0 포함 확인
+    if (input.contains('0')) throw IllegalArgumentException("input string should not contains '0'")
+    // 숫자 변환 가능 여부
+    if (input.toIntOrNull() == null) throw IllegalArgumentException("input string is not parseable")
+    // 중복값 확인
+    if (isDuplicated(input)) throw IllegalArgumentException("number is duplicated in input string")
+}
+
+// 중복 검사(사용자 숫자)
+fun isDuplicated(input: String): Boolean {
+    input.forEachIndexed { index, num ->
+        var count = 0
+        for (j in index..input.lastIndex) {
```

Comment:
```suggestion
        for (j in input.indices) {
```
indices라는 메서드를 사용하면 간단해 집니다.

Reply:
@Iwillbeagood 말씀해주신대로 하면 보다 간결한 코드를 작성할 수 있을것 같아요. 감사합니다! 👍